README.md

```diff
@@ -208,6 +208,21 @@ And then:
 
     go get -u github.com/shenwei356/seqkit/seqkit
 
+#### Method 4: Docker based installation
+    [Install Docker](https://docs.docker.com/engine/installation/#supported-platforms)
+    git clone this repo:
+    `git clone https://github.com/shenwei356/seqkit`
+
+    Run the following commands:
+    ```
+    cd seqkit
+    docker build -t shenwei356/seqkit .
+    docker run -it -p 8888:8888 shenwei356/seqkit:latest
+    go build
```

Comment:
`go get -u github.com/shenwei356/seqkit/seqkit` has built seqkit, and seqkit is ready to run. 

For test:

    cd /go/src/github.com/shenwei356/seqkit
    ./tests/test.sh

Reply:
when executing ./tests/test.sh binary could not be found

Reply:
I tested on a fresh vps, it can be found, cause

    ENV PATH $PATH:$GOPATH/bin:/usr/local/go/bin